app/models/book.rb

```diff
@@ -16,4 +16,17 @@ class Book < ActiveRecord::Base
   def reviewed_by?(user)
     reviews.exists?(user: user)
   end
+
+  def update_avg_rating
+    sum = 0
+    count = 0
+    reviews.each do |review|
```

Comment:
They whole can be replaced with `update(average_rating, reviews.average(:rating))`